jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -475,6 +479,10 @@ def _on_viewer_change(self, msg={}):
         if hasattr(self, 'viewer') and ref_data is not None:
             if ref_data.label in self.orientation.choices:
                 self.orientation.selected = ref_data.label
+            else:
+                self.orientation.selected = "Default orientation"
+                # Need to manually trigger this here for...reasons
+                self._change_reference_data()

```

Comment:
If "reasons" is unknown, we have to be sure this won't accidentally break #2751 😬

Reply:
I rebased onto your #2751 branch and both our fixes still work.